File: TransCoder_tokenized_test_set_functions/java/COORDINATES_RECTANGLE_GIVEN_POINTS_LIE_INSIDE.java
static void printRect ( Integer X [ ], Integer Y [ ], int n ) {
  int Xmax = Collections . max ( Arrays . asList ( X ) );
  int Xmin = Collections . min ( Arrays . asList ( X ) );
  int Ymax = Collections . max ( Arrays . asList ( Y ) );
  int Ymin = Collections . min ( Arrays . asList ( Y ) );
  System . out . println ( "{" + Xmin + ", " + Ymin + "}" );
  System . out . println ( "{" + Xmin + ", " + Ymax + "}" );
  System . out . println ( "{" + Xmax + ", " + Ymax + "}" );
  System . out . println ( "{" + Xmax + ", " + Ymin + "}" );
}

File: TransCoder_tokenized_test_set_functions/python/COORDINATES_RECTANGLE_GIVEN_POINTS_LIE_INSIDE.py
def printRect ( X, Y, n ) :
    Xmax = max ( X )
    Xmin = min ( X )
    Ymax = max ( Y )
    Ymin = min ( Y )
    print ( "{", Xmin, ", ", Ymin, "}", sep = "" )
    print ( "{", Xmin, ", ", Ymax, "}", sep = "" )
    print ( "{", Xmax, ", ", Ymax, "}", sep = "" )
    print ( "{", Xmax, ", ", Ymin, "}", sep = "" )
